Opserver.Core/Data/Dashboard/Providers/WmiDataProvider.Polling.cs

```diff
@@ -310,6 +310,65 @@ FROM Win32_LogicalDisk
                 }
             }
 
+            private async Task GetServicesAsync()
+            {
+                if (!String.IsNullOrEmpty(ServicesPatternRegEx?.ToString()))
```

Comment:
I would run this regardless (we can add a new bool to settings if needed later), and only filter if the regex is present.